tests/test_recreation.py

```diff
@@ -1125,9 +1120,9 @@ def test_year_order(self):
                 SAMPLE_DATA, 'predictors_scenario.csv'),
             'workspace_dir': self.workspace_dir,
         }
-
-        with self.assertRaises(ValueError):
-            recmodel_client.execute(args)
+        msgs = recmodel_client.validate(args)
+        self.assertEqual(
+            'Start year must be less than or equal to end year.', msgs[0][1])

```

Comment:
Depending on my other comment, we may also still want to check for `assertRaises(ValueError)`